scripts/lib/CIME/case_run.py

```diff
@@ -244,7 +257,7 @@ def case_run(case, skip_pnl=False):
            "You are not calling the run script via the submit script. "
            "As a result, short-term archiving will not be called automatically."
            "Please submit your run using the submit script like so:"
-           " ./case.submit")
+           " ./case.submit Time: {}".format(get_timestamp()))
```

Comment:
Is this change related to multi_driver? If not, please open an issue and submit it as a PR.